app/controllers/exercises_controller.rb

```diff
@@ -30,7 +30,9 @@ def start!
   end
 
   def set_default_content
+    @file_content = @exercise.files_content_for(current_user).except('content')
```

Comment:
Also, the attribute and method names are not consistent. The should be `@file_contents` and `file_contents_for`

Reply:
done